Tests/scripts/update_id_set.py

```diff
@@ -348,13 +371,65 @@ def process_testplaybook_path(file_path):
     print("adding {0} to id_set".format(file_path))
     script = None
     playbook = None
-    if re.match(TEST_SCRIPT_REGEX, file_path, re.IGNORECASE):
+    if checked_type(file_path, (TEST_SCRIPT_REGEX, PACKS_TEST_PLAYBOOKS_REGEX)):
```

Comment:
or whatever the correct name is...
```suggestion
    if checked_type(file_path, (TEST_SCRIPT_REGEX, PACKS_TEST_SCRIPT_REGEX)):
```